tests/types/Engine.Heaps.txt

```diff
@@ -0,0 +1,3 @@
+ng32/steam.hdll
```

Comment:
This shouldn't be here because it has no regex (even though it currently works).